tests/model_serving/model_server/llmd/llmd_configs/config_singlenode_prefill_decode.py

```
@@ -0,0 +1,85 @@
+    @classmethod
+    def container_env(cls):
+        return super().container_env() + [
+            {
+                "name": "VLLM_ADDITIONAL_ARGS",
+                "value": '--kv_transfer_config \'{"kv_connector":"NixlConnector","kv_role":"kv_both"}\'',
+            },
+            {
+                "name": "VLLM_NIXL_SIDE_CHANNEL_HOST",
+                "valueFrom": {"fieldRef": {"fieldPath": "status.podIP"}},
+            },
+        ]
```

Ruff (0.15.12)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)